app/Resources/translations/messages.en.xlf

```diff
@@ -276,6 +276,14 @@
                 <source>post.deleted_successfully</source>
                 <target>Post deleted successfully!</target>
             </trans-unit>
+            <trans-unit id="post.comment_added">
```

Comment:
@yceruto, I think you're right! I've changed messages names and added a more appropriate prefix for them.